cmd/oci-catalog/Cargo.toml

```diff
@@ -12,7 +12,7 @@ env_logger = "0.10"
 futures-core = "0.3"
 log = "0.4"
 prost = "0.11"
-reqwest = "0.11"
+reqwest = { version = "0.11", default-features = false, features = ["rustls-tls"] }
```

Comment:
Struggled to avoid the run-time error:
```
/oci-catalog/oci-catalog: error while loading shared libraries: libssl.so.1.1: cannot open shared object file: No such file or directory
```
so the above switches off the default `reqwest` feature which enables native tls and uses rustls-tls exclusively.

Reply:
Ah, makes sense, latest debian distro deprecated `libssl.so.1.1` in favor of a newer version, nice to see it's doable to fall back to rustls-tls. Thanks for the info!

Reply:
> Ah, makes sense, latest debian distro deprecated `libssl.so.1.1` in favor of a newer version

Yeah, but I even tried compiling with `rust:1.71.1-bookworm` , which compiled fine (so I assume should have been linking to libssl3), but it would have the same runtime error (tested in that same build image just running binary with `-h`, rather than transferring to the output image). But yes, glad we can use rusttls instead.